src/Parser.php

```diff
@@ -1281,6 +1281,13 @@ protected function isPlainCssValidElement($parsed, $allowExpression = false)
             case Type::T_MEDIA:
                 return $parsed;
 
+
+            case Type::T_CUSTOM_PROPERTY:
+                if ($parsed[2][2][1][0] == Type::T_INTERPOLATE) {
```

Comment:
Please use `$this->isPlainCssValidElement($parsed[2])` instead of hardcoding a particular structure (which might not be valid depending on how it looks like).

Reply:
also, this detects only one kind of SCSS syntax.

Reply:
ah thanks, that makes much more sense